src/cookbooks/publishing_datasets.md

```diff
@@ -1,13 +1,13 @@
 # Making Datasets Publicly Available
 
 Currently, only datasets and query results that are available in BigQuery and
-defined in the [bigquery-etl][bigquery_etl] repository can be made publicly available.
-See [Scheduling BigQuery Queries in Airflow](../cookbooks/bigquery-airflow.md)
-on how to create and schedule datasets. Before data can be published, a data review is
+defined in the [BigQuery-ETL][bigquery_etl] repository can be made publicly available.
```

Comment:
I'm of the opinion that styling should be `bigquery-etl` rather than BigQuery-ETL, as it refers to a technical artifact. Definitely not a hill I'm going to die on.

Reply:
I think I went with this spelling because my tutorial used it. But I don't think I used that capitalization there very consciously... happy to change it.